src/main/resources/META-INF/plugin.xml

```diff
@@ -202,6 +202,32 @@
                     description="Restarts the current application and attach the debugger">
             </action>
             <add-to-group group-id="AndroidToolsGroup" anchor="first"/>
+            <separator/>
+
+            <action id="com.developerphil.adbidea.action.EnableWifiAction"
+                    class="com.developerphil.adbidea.action.EnableWifiAction"
+                    text="Enable Wi-Fi"
+                    description="Enable Wi-Fi on device or emulator">
+            </action>
+
+            <action id="com.developerphil.adbidea.action.DisableWifiAction"
+                    class="com.developerphil.adbidea.action.DisableWifiAction"
+                    text="Disable Wi-Fi"
+                    description="Disable Wi-Fi on device or emulator">
+            </action>
+
+            <action id="com.developerphil.adbidea.action.EnableMobileAction"
+                    class="com.developerphil.adbidea.action.EnableMobileAction"
+                    text="Enable Mobile Data"
```

Comment:
Every other actions start with `ADB` so that they were grouped together in `find actions` and in the keymap settings.
That doesn't seem to be necessary anymore but I'd rather keep it for now to maintain the same standard.
And we can probably open another PR to remove the ADB prefix everywhere afterwards.

Reply:
Sure, fixed, I'll push both the fixes as soon as they are ready :)